project1/templates/register.html

```diff
@@ -63,34 +63,55 @@
 
 {% block body %}
     {% if act == 0 %}
+    <div class="registration">
+        <form action = "{{ url_for('register') }}" method = "POST">
+            <div class="container">
+                    <h1>Register</h1>
+                    <p>Please fill in this form to create an account.</p>
+                    <hr>
+
+                    <label for="usr"><b>Username</b></label>
+                    <input type="text" placeholder="Enter Username" name="usr" required>
+
+                    <label for="psw"><b>Password</b></label>
+                    <input type="password" placeholder="Enter Password" name="psw" required>
+
+                    <hr>
+                    <input name="but" type = "submit" value = "Login" class="btn">
+                    <input name="but" type = "submit" value = "Register" class="btn">
```

Comment:
Both login and register have the same name - "but". Are you able to differentiate? Please test it thoroughly.